src/Overcrowdin/CommandUtilities.cs

```diff
@@ -30,29 +29,10 @@ public static Credentials GetCredentialsFromConfiguration(IConfiguration config)
 				return null;
 			}
 
-			var userNameEnvVar = config["user_identifier_env"];
-			var userName = "";
-			if (string.IsNullOrEmpty(userNameEnvVar))
-			{
-				Console.WriteLine("The Crowdin configuration file is missing or did not contain 'user_identifier_env' " +
-								"(the environment variable containing your Crowdin username).");
-				// Don't fail; some grandfathered projects don't need the username
-			}
-			else
-			{
-				userName = Environment.GetEnvironmentVariable(userNameEnvVar);
-				if (string.IsNullOrEmpty(userName))
-				{
-					Console.WriteLine($"Environment variable {userNameEnvVar} did not contain your Crowdin username.");
-					return null;
-				}
-			}
+			var branch = string.IsNullOrEmpty(optionsBranch) ? config["branch"] : optionsBranch;
 
-			if (string.IsNullOrEmpty(userName))
-			{
-				return new ProjectCredentials {ProjectKey = apiKey};
-			}
-			return new AccountCredentials { AccountKey = apiKey, LoginName = userName };
+			var settings = await CrowdinProjectSettings.Init(config["project_identifier"], branch, apiKey, apiFactory);
+			return settings;
 		}
 
 		public static void GetFileList<T>(IConfiguration config, IFileOptions opts, IFileSystem fs,
```

Comment:
I don't see the folders parameter being used anywhere this method is called, it could probably be removed.